dagmc/dagnav.py

```diff
@@ -90,6 +90,25 @@ def geom_dimension_tag(self):
             create_if_missing=True,
         )
 
+    def add_volume_groups(self, group_map):
+        """Adds groups of volumes to the model.
+
+        Parameters
+        ----------
+        group_map : dict
+            A dictionary mapping group names to a 2-tuple of (int, list) whose
+            first element is the group ID and the second element is a list of
+            volume IDs or Volume objects in the group.
```

Comment:
Since groups contain either surfaces or volumes in a DAGMC model, what do you think about a generic function for generating a groups of surfaces. Probably less common that it would be used compared to the volume version.. idk. Thoughts?

Reply:
I agree and think the functionality to add multiple `DagSet`s to a Group should probably live in the `Group` class. I just hadn't yet worked through exactly what the handles are and how to deal with that, but if you think an `add_sets` (rather than `add_set`) method or `add/set_volumes/surfaces` type methods would be valuable on the `Group` class I will go that route and use that functionality in the model level `add_volume_groups` method instead.

Reply:
Yeah let's start down that route, but if you hit any major road blocks don't spend too much time worrying about it at this point?